### PR TITLE
[Content] New documentation site about pages

### PR DESCRIPTION
**Pull request by ronijaakkola**
## Description
This PR adds about pages to the new documentation site. This includes:
- Support
- Contribution
- Accessibility (statement)
- What is new
- Roadmap
- Resources

All pages have been run through a grammar check too.

## Related Issue
https://helsinkisolutionoffice.atlassian.net/browse/HDS-1184

## How Has This Been Tested?
Tested by running the new documentation site locally.

### CODE REVIEW

File: new-site/src/docs/about/contributing/before-contributing.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Depending on the feature you want to be working on, you will need access to at least some of these tools:
+- <Link size="M" href="https://www.sketch.com/" external>Sketch</Link>
+- City of Helsinki organisation in <Link size="M" href="https://www.abstract.com/" external>Abstract</Link>
+- City of Helsinki Slack (channel: <Link size="M" href="https://helsinkicity.slack.com/archives/CHCV3KTHA" external>#designsystem</Link>)
+- <Link size="M" href="https://github.com/City-of-Helsinki/helsinki-design-system" external>HDS GitHub repository</Link>
+
+## Contribution process
+When faced with a need for a component or feature, you can ask yourself a few simple questions to decide whether you should propose a new component or feature to be added in HDS, or design a custom implementation:
+
+<Image src="../../../hds-contribution.png" alt="How to contribute to Helsinki Design System" style="max-width:100%" viewable />
```
Comment (minevala) [inline new-site/src/docs/about/contributing/before-contributing.mdx:39]: The image urls do not work. I tried to solve this but maybe they have to be fixed in another PR

Comment (ronijaakkola) [reply new-site/src/docs/about/contributing/before-contributing.mdx:39]: Images are not passed through to the `/public` folder. That's seems to be the reason.

**Review by minevala — APPROVED** [review]
Looks about the same as in the old documentation 😁 
Let's fix images separately.
